Clinical-Genomics/scout
# Script to fix OMIM format for old cases

I think the easiest way to fix #3301 would be with a scripts that loops over the cases with the old OMIM format and fixes them one by one. To be first tested on stage

**How to test**:
1. On hasta, create a file with the content of the script.
2. "us" to use scout stage env (python and other libs are required)
3. Run a simulation to check which cases contain wrong diagnosis format: `python convert_case_omim_format.py --db_uri stage_db_uri`
- [x] Make sure that there are cases with old diagnosis format (a list of integers). These cases won't be modified by the script
4. Run the script to fix the stage cases with old  diagnosis format: `python convert_case_omim_format.py --db_uri stage_db_uri --fix`
- [x] These cases should be fixed. You can make sure by running the command `python convert_case_omim_format.py --db_uri stage_db_uri`, that will no longer show any cases with old dia now.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
